dgraph/sample/data.json

```diff
@@ -14,23 +14,219 @@
           "value": "6.2.1 Beta-lactam medicines"
         },
         {
-          "type": "unspc_code",
-          "value": "51281712"
+          "type": "drugbank_code",
+          "value": "DB01060",
+          "has_property": [
+            {
+              "type": "source",
+              "value": "https://drugbank.ca/drugs/" 
+            }
+          ]
         },
         {
-          "type": "usfda_code",
-          "value": "544Y3K6MYH"
+          "type": "atc_code",
+          "value": "J01CA04"
         },
         {
-          "type": "drugbank_code",
-          "value": "DB01060"
+          "type": "synonym",
+          "value": "Amoxicillin Anhydrous"
         },
         {
-          "type": "atc_code",
-          "value": "J01CA04"
+          "type": "synonym",
+          "value": "Amoxicillin trihydrate",
+          "has_property": [
+            {
+              "type": "unspc_code",
+              "value": "51281712"
+            }
+          ]
+        },
+        {
+          "type": "synonym",
+          "value": "Amoxicillin sodium",
+          "has_property": [
+            {
+              "type": "usfda_code",
+              "value": "544Y3K6MYH"
+            }
+          ]
         }
       ],
-      "has_child": []
+      "has_child": [
+        {
+          "description": "Amoxicillin oral",
+          "type": "form_category",
+          "has_property": [
+            {
+                "value": "350162003",
+                "type": "SnoMED CT"
+            }
+          ],
+          "has_child": [
+            {
+              "description": "Amoxicillin oral capsule",
+              "type": "form",
```

Comment:
Same as above :).

Reply:
Added in 622f2d4724afccb5ae04556103a8e5b003b21c90 (code choice may not have been completely arbitrary) 😸

Reply:
I'd recently been made aware of how the current codes are generated: https://wiki.sussol.net/doku.php/projects:universal_drug_codes:accessing_rds_instance?s[]=universal&s[]=codes#add_a_code_new_universal_code

It's a substring of the UUID used in mSupply for the item. Quite likely not helpful here, as these will be new nodes. Just mentioning because I thought it interesting.